syntax = "proto3";
package strategy;




import "easy_flow_sdk/model/easy_flow/deploy_strategy.proto";

/*
Get请求
*/
message GetRequest {
    /*
    策略Id
    */
    string strategyID = 1 ;
}

/*
Get返回
*/
message GetResponse {
    /*
    错误码
    */
    int32 code = 1 ;
    /*
    消息
    */
    string msg = 2 ;
    /*
    返回数据
    */
    easy_flow.DeployStrategy data = 3 ;
}

/*
GetApi返回
*/
message GetResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    strategy.GetResponse data = 4 ;
}

